client/components/domains/use-my-domain/utilities/get-option-info.js

```diff
@@ -126,6 +126,36 @@ export function getOptionInfo( {
 				),
 			};
 			break;
+		case domainAvailability.RECENT_REGISTRATION_LOCK_NOT_TRANSFERRABLE:
+			transferContent = {
+				...optionInfo.transferNotSupported,
+				topText: createInterpolateElement(
+					sprintf(
+						/* translators: %s - the domain the user wanted to transfer */
+						__(
+							"<strong>%s</strong> can't be transferred because it was registered less than 60 days ago, but you can connect it instead."
+						),
+						domain
+					),
+					{ strong: createElement( 'strong' ) }
+				),
+			};
+			break;
+		case domainAvailability.SERVER_TRANSFER_PROHIBITED_NOT_TRANSFERRABLE:
+			transferContent = {
+				...optionInfo.transferNotSupported,
+				topText: createInterpolateElement(
+					sprintf(
+						/* translators: %s - the domain the user wanted to transfer */
+						__(
+							"<strong>%s</strong> can't be transferred due to a domain lock, but you can connect it instead."
```

Comment:
To help the user understand what's going on here, I wonder if we should be more specific and say something like "...due to a transfer lock at the registry, ..."

Reply:
I like that 👍 it should make things a little more clear.